code/modules/client/preferences/migrations/tgui_prefs_migration.dm

```diff
@@ -70,6 +70,8 @@
 		"random_undershirt" = "undershirt",
 		"random_underwear" = "underwear",
 		"random_underwear_color" = "underwear_color",
+		"random_bra" = "bra", //SKYRATS EDIT ADDITION
+		"random_bra_color" = "bra_color", //SKYRATS EDIT ADDITION
```

Comment:
You don't need to do this, there's no migration for randomization settings that didn't exist.